### ISSUE TITLE
Documenting subtasks?

### ISSUE THREAD

**Issue by dotherightthing** [ben-eb/grunt-available-tasks#22]
Does the plugin support this?

The screenshot in your documentation shows:

    clean -> Clean files and folders. (dist | tmp | templates)

Is the text in brackets documentation of the available sub tasks ? ie `clean:dist`, `clean:tmp` and `clean:templates`

If so, how do I configure the plugin to include information about these sub tasks?

Thanks.

**Comment by ben-eb** [comment]
Yes, that is for showing the multitask targets. There isn't any additional information about these that we can get from Grunt. But we could introduce a `multiTasks` option to add descriptions for them.

Would be open to a PR for this. I'm pretty busy on other things right now so this isn't a high priority for me.
